utils/api/services.js

```diff
@@ -2,7 +2,7 @@ import useSWR from 'swr'
 
 /** GET METHODS */
 export const useAllWares = (accessToken) => {
-  const { data, error } = useSWR([`/providers/${process.env.NEXT_PUBLIC_PROVIDER_ID}/wares.json`, accessToken])
+  const { data, error } = useSWR([`/wares.json`, accessToken])
```

Comment:
now that we're using an org that has a single provider, unlike acme, we don't need to filter by provider.